SDK/AppCenter/Microsoft.AppCenter.Shared/WrapperSdk.cs

```diff
@@ -3,7 +3,7 @@
 
 namespace Microsoft.AppCenter
 {
-    public static class WrapperSdk
+    public partial class WrapperSdk
     {
         public const string Name = "appcenter.xamarin";
```

Comment:
Oh interesting, this should be moved to a Xamarin shared code project, this does not have to be done in this PR though.